lib/Basics/HybridLogicalClock.h

```diff
@@ -123,7 +122,9 @@ class HybridLogicalClock {
   /// the result buffer are returned
   static std::pair<size_t, size_t> encodeTimeStamp(uint64_t t, char* r) {
     size_t pos = 11;
+    r[pos - 1] = '\0';
```

Comment:
I do not understand this. If t is equal to 0, we return the pair 11, 0 anyway. If t is not equal to 0, then this ``\0'` is overwritten in the first run of the loop. Anyway, it also does no harm, as far as I can see.

Reply:
Ah, true.

Reply:
Fixed.